core/src/banking_stage.rs

```diff
@@ -804,6 +804,9 @@ impl BankingStage {
         packet_indexes: Vec<usize>,
     ) {
         if !packet_indexes.is_empty() {
+            if unprocessed_packets.len() > 400 {
```

Comment:
@pgarg66 I found this has much better memory characteristics for fullnodes with capping the buffered packets size, even with the original code. Any concerns?